panel/layout/base.py

```diff
@@ -817,15 +835,16 @@ def _linked_properties(self):
         )
 
     def _process_param_change(self, params: Dict[str, Any]) -> Dict[str, Any]:
-        if 'scroll' in params:
+        if params.get('scroll'):
```

Comment:
With the change, you no longer run the if statement if `scroll` is False. Is that deliberate?

If it is, you could use a walrus operator and combine it with the next line.

Reply:
> With the change, you no longer run the if statement if scroll is False. Is that deliberate?

Yes, setting `scroll=False` didn't do anything I think.